src/captureScreenshot.js

```diff
@@ -129,7 +129,7 @@ exports.handler = async (event, callback) => {
     return buffer;
   });
 
-  const url = await S3Bucket.upload(buffer, imageType, options.s3key);
+  const response = await S3Bucket.upload(buffer, imageType, options.s3key);
 
-  return { url };
+  return JSON.stringify(response);
```

Comment:
Not sure why you wanted to return a string instead of the object?

Reply:
This was an artifact from another PR I have coming shortly to support private API Gateway request/response, but yes if not a response back to API Gateway, it should be an object. Good Catch!